src/cockpit/389-console/src/lib/security/securityModals.jsx

```diff
@@ -10,35 +10,44 @@ import {
     GridItem,
     Modal,
     ModalVariant,
+    Radio,
     Text,
+    TextArea,
     TextContent,
     TextVariants,
     TextInput,
+    Tooltip,
     ValidatedOptions,
 } from "@patternfly/react-core";
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/js/icons/outlined-question-circle-icon';
 import PropTypes from "prop-types";
 
-export class SecurityAddCACertModal extends React.Component {
+export class ExportCertModal extends React.Component {
     render() {
         const {
             showModal,
             closeHandler,
             handleChange,
             saveHandler,
+            fileName,
+            nickName,
+            binaryFormat,
             spinning,
-            error
         } = this.props;
-        let saveBtnName = "Add Certificate";
+
+        let exportBtnName = "Export Certificate";
         const extraPrimaryProps = {};
         if (spinning) {
-            saveBtnName = "Adding Certificate ...";
-            extraPrimaryProps.spinnerAriaValueText = "Saving";
+            exportBtnName = "Exporting Certificate ...";
+            extraPrimaryProps.spinnerAriaValueText = "Exporting";
         }
 
+        const title = <>Export Certificate: &nbsp;&nbsp;<i>{nickName}</i></>;
```

Comment:
When we do 'Export Certificate', it's not clear to me where it exports and how a user can find it later.
Maybe we should add something like 'Backup List'? Or, at least, report the cert path where the user can find the file.

And when we import the certificate, should we add something like 'file browser' button? I don't have a strong opinion here though...

Reply:
> When we do 'Export Certificate', it's not clear to me where it exports and how a user can find it later. Maybe we should add something like 'Backup List'? Or, at least, report the cert path where the user can find the file.

Let me think about this, but yes it needs to be improved.

> 
> And when we import the certificate, should we add something like 'file browser' button? I don't have a strong opinion here though...

Isn't the file browser only for the client machine?  The import needs to be a file local to the server.  I could list certs that are already present in the nssalpd-certdir (/etc/dirsrv/slapd-inst).

Reply:
> Isn't the file browser only for the client machine? The import needs to be a file local to the server. I could list certs that are already present in the nssalpd-certdir (/etc/dirsrv/slapd-inst).

Ideally, I think we need a way to import the certs from nssalpd-certdir and from the client machine.
The second is possible but only with the cert copy past. And I think it'll be nice to offer an option with a file browser. Probably, something like this (FileUpload component):
https://github.com/389ds/389-ds-base/blob/main/src/cockpit/389-console/src/lib/ldap_editor/lib/editableTable.jsx#L824

Reply:
@droideck - all request features implemented, please review...